js/containers/DeployHistory.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var ReactRedux = require('react-redux');
+var actions = require('../_actions.js');
+
+var DeployHistory = function(props) {
+	var tableRows = [];
+
+	for (var i in props.data) {
+		var row = props.data[i];
+		tableRows.push(
+			<tr key={i}>
+				<td>{row.CreatedDate}</td>
+				<td>{row.CommitMessage}</td>
+				<td>{row.Deployer}</td>
+				<td>{row.State}</td>
+			</tr>
+		);
+	}
+
+	return (
+		<div>
+			<h4>History</h4>
+			<table className="table table-striped table-hover">
+				<thead>
+					<tr>
+						<th>Request date</th>
+						<th>Build details</th>
+						<th>Deployed by</th>
+						<th>Status</th>
+					</tr>
+				</thead>
+				<tbody>
+					{tableRows}
+				</tbody>
+			</table>
+		</div>
+	);
+};
+
+const mapStateToProps = function(state) {
+	return {
+		data: state.deployhistory.data
+	};
+};
+
+const mapDispatchToProps = function(dispatch) {
+	dispatch(actions.getDeployHistory());
```

Comment:
this should not be here, this section is used for sending props that are callbacks for dispatching actions. For example a refresh button / link.

See Plan.jsx `store.dispatch(actions.getRevisionsIfNeeded());`